packages/strapi-plugin-old-slugs/README.md

```diff
@@ -0,0 +1,3 @@
+# Strapi plugin strapi-plugin-old-slugs
+
+A quick description of strapi-plugin-old-slugs.
```

Comment:
Non blocking: iets meer context in de beschrijving zou handig zijn voor maintainability in de toekomst. Gewoon kort wat doet de plugin, en hoe?